shared/domains/chelonia/utils.js

```diff
@@ -236,7 +236,9 @@ export const validateKeyUpdatePermissions = (contractID: string, signingKey: SPK
     if (uk.id && uk.id !== uk.oldKeyId) {
       updatedMap[uk.id] = uk.oldKeyId
     }
-    const updatedKey = { ...existingKey }
+    // Discard `_notAfterHeight` and `_notBeforeHeight`, since retaining them
+    // can cause issues reprocessing messages.
+    const { _notAfterHeight, _notBeforeHeight, ...updatedKey } = { ...existingKey }
```

Comment:
Question + request:

- Wouldn't this trigger an unused variable linter warning? (And if not... why not?)
- Could you elaborate on the comment a bit more and give maybe 1 example of an issue that would happen?

Reply:
Consider `omit` from `turtledash`?